project/AppDependencies.scala

```diff
@@ -1,13 +1,13 @@
-import sbt._
+import sbt.*
 
 object AppDependencies {
 
-  private val bootstrapVersion = "8.6.0"
+  private val bootstrapVersion = "9.0.0"
 
   val compile = Seq(
     // format: OFF
     "uk.gov.hmrc"     %% "bootstrap-frontend-play-30"                    % bootstrapVersion,
-    "uk.gov.hmrc"     %% "play-frontend-hmrc-play-30"                    % "10.0.0",
+    "uk.gov.hmrc"     %% "play-frontend-hmrc-play-30"                    % "10.1.0",
```

Comment:
since you're updating the BE at the same time we may as well merge the BE, let a new build version number build and then bring that updating version below where we've currently got:
```scala
"uk.gov.hmrc" %% "direct-debit-update-email-backend-cor-journey" % "0.17.0",
```